lib/ts/recipe/openid/types.ts

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import OverrideableBuilder from "supertokens-js-override";
+import { BaseRequest, BaseResponse } from "../../framework";
+import NormalisedURLDomain from "../../normalisedURLDomain";
+import NormalisedURLPath from "../../normalisedURLPath";
+import { RecipeInterface as JWTRecipeInterface, APIInterface as JWTAPIInterface, JsonWebKey } from "../jwt/types";
+
+export type TypeInput = {
```

Comment:
We should also give it `jwtValiditySeconds` since the jwt recipe also accepts that. So we can make this:
```
{
   issuer?: string,
   jwtFeature?: { // this will be the same in normalised version as well
      jwtValiditySeconds?: number
   },
   override?: {...}
}
```

Reply:
Done